packages/cf-style-container/package.json

```diff
@@ -12,9 +12,10 @@
   },
   "dependencies": {
     "fela": "^5.0.0",
-    "merge-options": "0.0.64",
+    "merge-options": "^1.0.0",
+    "prop-types": "^15.5.8",
     "react-fela": "^5.0.0",
-    "prop-types": "^15.5.8"
+    "underscore.string": "^3.3.4"
```

Comment:
What's capitalize being used for? It seems that the only reason for including underscore.string is to export it. Why not just import it where you need it?